### ISSUE TITLE
The web http://clueminer.org is down

### ISSUE THREAD

**Issue by jirutka** [deric/clueminer#18]

**Comment by deric** [comment]
Yeah, we'll put it back online with the first stable release, together with release to Maven central.

**Comment by jirutka** [comment]
Okay then. Do you wanna help with releasing to Maven Central?

**Comment by deric** [comment]
Guess I'm gonna try Sonatype OSS, it's sort of prepared for that.

**Comment by jirutka** [comment]
How it that related to clueminer.org website?

Sonatype OSS _is_ basically Maven Central. However, if you don’t wanna suffer so much, use Bintray as an (almost) [pain-free gateway to Maven Central](http://blog.bintray.com/2014/02/11/bintray-as-pain-free-gateway-to-maven-central). We have already organization account [cvut](https://bintray.com/cvut/) on Bintray.
